### ISSUE TITLE
✅ 🐞 Bug Report: Elementor 3.13: Self scrolling to table of contents in mobile view - [ED-10888]

### ISSUE THREAD

**Issue by Optitesting** [elementor/elementor#22381]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

After updating to Elementor 3.13, page in mobile view self scrolls down to table of contents when loading. 

The arrow/icon in table of contents upper field got a dark square around it. 

Only fix was to roll back to older Elementor version. 

### Steps to reproduce

Tried to empty all cache and regenerated Elementor css..
Tried to make changes in table of contents. 
Tried to deactivate plugins.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.3.38
	PHP Version: 8.1.18
	PHP Memory Limit: 4294967296
	PHP Max Input Vars: 5000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://www.optihus.se
	Home URL: https://www.optihus.se
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 1024M
	Max Memory limit: 4294967296
	Permalink Structure: /%postname%
	Language: sv-SE
	Timezone: Europe/Stockholm
	Admin Email: kontoret@optihus.se
	Debug Mode: Inactive

== Theme ==
	Name: GeneratePress
	Version: 3.3.0
	Author: Tom Usborne
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: sv_SE
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36

== Active Plugins ==
	Asset CleanUp Pro: Page Speed Booster
		Version: 1.2.3.1
		Author: Gabe Livan

	Calculated Fields Form
		Version: 1.1.163
		Author: CodePeople

	Checkout Field Editor for WooCommerce
		Version: 1.8.2
		Author: ThemeHigh

	Classic Editor
		Version: 1.6.3
		Author: WordPress Contributors

	Code Snippets
		Version: 3.3.0
		Author: Code Snippets Pro

	Cookie Notice & Compliance for GDPR / CCPA
		Version: 2.4.8
		Author: Hu-manity.co

	Custom Payment Gateways for WooCommerce
		Version: 1.7.0
		Author: Imaginate Solutions

	Custom Product Tabs for WooCommerce
		Version: 1.8.2
		Author: YIKES, Inc.

	Custom Product Tabs Pro
		Version: 1.2.4
		Author: YIKES, Inc.

	Elementor
		Version: 3.12.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.3
		Author: Elementor.com

	Google Listings and Ads
		Version: 2.4.4
		Author: WooCommerce

	GP Premium
		Version: 2.3.1
		Author: Tom Usborne

	Premmerce Permalink Manager for WooCommerce (Premium)
		Version: 2.3.8
		Author: premmerce

	Really Simple SSL
		Version: 6.2.5
		Author: Really Simple Plugins

	Seraphinite Accelerator (Full, premium)
		Version: 2.19.9
		Author: Seraphinite Solutions

	Smart Custom 404 error page [404page]
		Version: 11.4.3
		Author: Peter Raschendorfer

	WooCommerce
		Version: 7.6.1
		Author: Automattic

	WooCommerce Dynamic Pricing & Discounts
		Version: 2.4.5
		Author: RightPress

	WooDiscuz - WooCommerce Comments
		Version: 2.2.9
		Author: gVectors Team

	Woo Title Limit
		Version: 2.0.3
		Author: Dima W.

	Wordfence Security
		Version: 7.9.2
		Author: Wordfence

	WP Mail SMTP
		Version: 3.8.0
		Author: WP Mail SMTP

	Yoast SEO
		Version: 20.6
		Author: Team Yoast

	Yoast SEO: WooCommerce
		Version: 15.7
		Author: Team Yoast


== Must-Use Plugins ==
	Asset CleanUp Pro: Plugin Filtering
		Version: 1.0
		Author: Gabriel Livan


== Elementanvändning ==
	
	container : 0
		container : 1
		heading : 1
		image : 13
		text-editor : 62
		toggle : 10
		video : 1
	section : 7
		image : 2
		text-editor : 6
		toggle : 1
	wp-post : 24
		text-editor : 24
		toggle : 11
	wp-page : 57
		container : 1
		icon-box : 1
		text-editor : 108
		toggle : 37
	single-page : 2
		author-box : 1
		post-comments : 1
		post-info : 1
		posts : 1
		share-buttons : 1
		text-editor : 1
		theme-post-content : 1
		theme-post-featured-image : 1
		theme-post-title : 1


== Inställningar ==
	
	cpt_support: post, page, product
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	font_display: block


== Funktioner ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrationer ==
	
	facebook_sdk: Aktiv
	woocommerce: Aktiv


== Elementor-experiment ==
	Optimized DOM Output: Aktiv
	Förbättrad inläsning av filer: Aktiv
	Förbättrad CSS-laddning: Aktiv
	Inline Font Icons: Inaktiv som standard
	Tillgänglighetsförbättringar : Aktiv
	Ytterligare anpassade brytpunkter: Inaktiv
	admin_menu_rearrangement: Inaktiv som standard
	Flexbox Container: Aktiv
	Uppgradera Swiper-bibliotek: Inaktiv som standard
	Default to New Theme Builder: Aktiv som standard
	Editor Top Bar: Inaktiv som standard
	Landningssidor: Aktiv som standard
	Nested Elements: Inaktiv
	Uppskjuten inläsning av bakgrundsbilder: Inaktiv som standard
	Sidövergångar: Aktiv som standard
	Notes: Aktiv som standard
	Loop: Aktiv
	Formulärinlämningar: Aktiv som standard
	Scroll Snap: Aktiv som standard
	Meny: Inaktiv som standard


== Logg ==
	
Log: showing 20 of 322023-04-16 07:37:37 [info] elementor-pro::elementor_pro_updater Started 
2023-04-16 07:37:37 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-16 07:37:37 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-16 07:37:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.2',
)]
2023-04-16 07:37:37 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.2',
)]
2023-04-16 07:38:17 [info] elementor::elementor_updater Started 
2023-04-16 07:38:17 [info] Elementor/Upgrades - _on_each_version Start  
2023-04-16 07:38:17 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.1',
)]
2023-04-16 07:38:18 [info] Elementor/Upgrades - _on_each_version Finished 
2023-04-16 07:38:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.1',
)]
2023-04-16 07:38:19 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.1',
)]
2023-05-08 23:06:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.2',
  'to' => '3.13.0',
)]
2023-05-08 23:06:28 [info] elementor-pro::elementor_pro_updater Started 
2023-05-08 23:06:28 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-08 23:06:28 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-08 23:06:28 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.2',
  'to' => '3.13.0',
)]
2023-05-08 23:07:26 [info] elementor::elementor_updater Started 
2023-05-08 23:07:26 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-08 23:07:26 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-08 23:07:26 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.1',
  'to' => '3.13.0',
)]

PHP: showing 9 of 9PHP: 2023-03-27 03:53:29 [notice X 103901][/customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-27 09:55:28 [notice X 73][/customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/woocommerce/module.php::562] parse_str(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-30 22:19:59 [notice X 100][/customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(200): class type strpos()
#2: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(212): ElementorPro\Modules\LoopBuilder\Module -> is_editing_existing_loop_item()
#3: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(60): ElementorPro\Modules\LoopBuilder\Module -> is_loop_theme_builder()
#4: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/documents/loop.php(386): ElementorPro\Modules\LoopBuilder\Module -> __construct()
',
)]
PHP: 2023-03-30 22:19:59 [notice X 100][/customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(200): class type strpos()
#2: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(212): ElementorPro\Modules\LoopBuilder\Module -> is_editing_existing_loop_item()
#3: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(60): ElementorPro\Modules\LoopBuilder\Module -> is_loop_theme_builder()
#4: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/documents/loop.php(308): ElementorPro\Modules\LoopBuilder\Module -> __construct()
',
)]
PHP: 2023-04-07 18:33:58 [warning X 12][/customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-16 08:36:15 [notice X 10][/customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::204] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(204): class type strpos()
#2: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(216): ElementorPro\Modules\LoopBuilder\Module -> is_editing_existing_loop_item()
#3: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(60): ElementorPro\Modules\LoopBuilder\Module -> is_loop_theme_builder()
#4: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/documents/loop.php(406): ElementorPro\Modules\LoopBuilder\Module -> __construct()
',
)]
PHP: 2023-04-16 08:36:15 [notice X 10][/customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::204] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(204): class type strpos()
#2: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(216): ElementorPro\Modules\LoopBuilder\Module -> is_editing_existing_loop_item()
#3: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/module.php(60): ElementorPro\Modules\LoopBuilder\Module -> is_loop_theme_builder()
#4: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/loop-builder/documents/loop.php(328): ElementorPro\Modules\LoopBuilder\Module -> __construct()
',
)]
PHP: 2023-04-16 08:36:15 [warning X 5][/customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-05-08 23:10:37 [warning X 540][/customers/8/b/3/optihus.se/httpd.www/wp-content/plugins/elementor-pro/modules/woocommerce/module.php::381] Undefined array key 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 1 of 1JS: 2023-04-07 16:19:55 [error X 9][https://www.optihus.se/wp-includes/js/jquery/jquery.min.js?ver=3.6.1:2:51824] Cannot read properties of undefined (reading 'replace') 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Kompatibilitet inte specificerad

== Elementor Pro - Compatibility Tag ==
	

				
````

</details>

**Comment by nicholaszein** [comment]
Hello, Jerker (@Optitesting)! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by Optitesting** [comment]
May I ask when you can take a closer look at this issue? As it is now I can´t update page and use your new features and safety updates, because of the issue.

**Comment by norras69** [comment]
I have the same issue on desktop

**Comment by Optitesting** [comment]
> I have the same issue on desktop

Have you opened a ticket with Elementor support? I think their work with fixing bugs takes way too long time. Is it just me or do you agree?

**Comment by norras69** [comment]
> Have you opened a ticket with Elementor support? I think their work with fixing bugs takes way too long time. Is it just me or do you agree?

I did open a ticket and after some discussion, I had this response yesterday:

"...this is related to a known issue. Our developers are aware of this and are working on it. A full report has been created, and the priority of this task has been set high. This means it is already in the process of resolving this issue. Therefore the fix should be available in some of the following updates.

The current status is Waiting for Release, which means this has already been fixed, but still waiting to be included in the following Elementor update..."

**Comment by Optitesting** [comment]
@norras69 Thank you for telling... Then there is hope :)

**Comment by Optitesting** [comment]
@nicholaszein I can see 3.14 is released but does it fix this bug? When can we see a fix? @norras69

**Comment by norras69** [comment]
> @nicholaszein I can see 3.14 is released but does it fix this bug? When can we see a fix? @norras69

This 3.14 has fixed the issues on my website - all seems to be working as normal.

**Comment by nicholaszein** [comment]
Hello guys! Apologies for the delay. 🙏 

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.14.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
